### DISCUSSION TITLE
Needs  Toolbar Pin/Unpin Functionality or Monitor Switching Shortcut Keys

### DISCUSSION THREAD

**Discussion by bradleytchu** [rustdesk/rustdesk#8335] [General]
I know this bug has been posted before but it needs attention, because it's worth noting that the icon was there for a reason. I have multiple monitors and I need to access to the toolbar to switch between monitors. The issue is the toolbar is covering up tabs in VS code and Chrome. 

Can we have the toolbar be movable or have shortcut keys for switching between monitors?

![Screenshot 2024-06-12 140147](https://github.com/rustdesk/rustdesk/assets/44756992/b9f7244d-11de-47db-abf3-8d08af2bc3d5)
![Screenshot 2024-06-12 140433](https://github.com/rustdesk/rustdesk/assets/44756992/179ee37a-8feb-4206-b597-d6d7ab7754f2)

**Comment by rustdesk** [comment]
duplicate
